src/nifake/metadata/functions.py

```diff
@@ -1407,7 +1425,7 @@
         'method_templates': [
             {
                 'documentation_filename': 'lock',
-                'library_interpreter_filename': 'default_method',
+                'library_interpreter_filename': 'lock',
```

Comment:
await: Re-copy from new export after [AB!377348](https://ni.visualstudio.com/DevCentral/_git/ni-central/pullrequest/377348) is completed.

Reply:
Done